aviary/mission/gasp_based/ode/taxi_eom.py

```diff
@@ -52,8 +61,14 @@ def compute(self, inputs, outputs):
         outputs[Dynamic.Mission.MASS] = takeoff_mass - outputs["taxi_fuel_consumed"]
 
     def compute_partials(self, inputs, J):
+        nn = self.options['num_nodes']
+
         dt_taxi = self.options['aviary_options'].get_val(Mission.Taxi.DURATION, 's')
 
-        J["taxi_fuel_consumed", Dynamic.Mission.FUEL_FLOW_RATE_NEGATIVE_TOTAL] = -dt_taxi
+        J["taxi_fuel_consumed", Dynamic.Mission.FUEL_FLOW_RATE_NEGATIVE_TOTAL] = \
+            -np.ones(nn) * dt_taxi
```

Comment:
You can actually change this to:
    
    J["taxi_fuel_consumed", Dynamic.Mission.FUEL_FLOW_RATE_NEGATIVE_TOTAL][:] = -dt_taxi

to save some math operations by taking advantage of numpy's ability to broadcast a scalar into slices.

Reply:
Got it. Done.